Userland/Libraries/LibWeb/Layout/Node.cpp

```diff
@@ -666,6 +667,15 @@ JS::NonnullGCPtr<NodeWithStyle> NodeWithStyle::create_anonymous_wrapper() const
     return *wrapper;
 }
 
+void NodeWithStyle::reset_table_box_computed_values_used_by_wrapper_to_init_values()
+{
+    VERIFY(is<TableBox>(*this));
+    static_cast<CSS::MutableComputedValues&>(m_computed_values).set_position(CSS::Position::Static);
+    static_cast<CSS::MutableComputedValues&>(m_computed_values).set_float(CSS::Float::None);
+    static_cast<CSS::MutableComputedValues&>(m_computed_values).set_inset({ CSS::Length::make_auto(), CSS::Length::make_auto(), CSS::Length::make_auto(), CSS::Length::make_auto() });
+    static_cast<CSS::MutableComputedValues&>(m_computed_values).set_margin({ CSS::Length::make_px(0), CSS::Length::make_px(0), CSS::Length::make_px(0), CSS::Length::make_px(0) });
```

Comment:
Instead of doing `static_cast` over and over, please put this in a local:
```cpp
auto& mutable_values = static_cast<CSS::MutableComputedValues&>(wrapper_computed_values);
mutable_values.set...
mutable_values.set...
```